src/Middleware/src/Headstart.API/Commands/EnvironmentSeedCommand.cs

```diff
@@ -514,15 +514,22 @@ public async Task CreateSecurityProfiles(EnvironmentSeed seed, string accessToke
 					Roles = p.Roles
 				}).ToList();
 
-			profiles.Add(new SecurityProfile()
-			{
-				Roles = new List<ApiRole> { ApiRole.FullAccess },
-				Name = _fullAccessSecurityProfile,
-				ID = _fullAccessSecurityProfile
-			});
+            profiles.Add(new SecurityProfile()
+            {
+                Roles = new List<ApiRole> { ApiRole.FullAccess },
+                Name = _fullAccessSecurityProfile,
+                ID = _fullAccessSecurityProfile
+            });
+			Console.WriteLine(profiles);
 
-			var profileCreateRequests = profiles.Select(p => _oc.SecurityProfiles.CreateAsync(p, accessToken));
-			await Task.WhenAll(profileCreateRequests);
+			var profileCreateRequests = profiles.Select(p => _oc.SecurityProfiles.SaveAsync(p.ID, p, accessToken));
+			try
+            {
+				await Task.WhenAll(profileCreateRequests);
+			} catch (Exception ex)
+            {
+				Console.WriteLine(ex);
```

Comment:
not sure if you wanted to keep the console.writelines in, but we'd probably want it logged elsewhere too for troubleshooting.